ElastosDIDSDK/DIDDocument.swift

```diff
@@ -687,34 +722,69 @@ public class DIDDocument {
         return DIDDocumentBuilder(self)
     }
 
-    public func sign(using storePassword: String, for data: Data...) throws -> String {
-        return try sign(self.defaultPublicKey, storePassword, data)
+    public func sign(withId: DIDURL, using storepass: String, for data: Data...) throws -> String {
```

Comment:
Why want to change prototype of sign method ? And sure to  remove sign(using storePassword: String ...)